src/safe_apps/tests/test_views.py

```diff
@@ -6,7 +6,7 @@
 
 class EmptySafeAppsListViewTests(APITestCase):
     def test_empty_set(self):
-        url = reverse("v1:safe-apps")
+        url = reverse("safe-apps:safe-apps")
```

Comment:
wouldn't something like `safe-apps:list_v1` be nicer?

Reply:
Yes I agree! 👍

Reply:
I'm going to do this change for the `chains:chains` and revert the changes on the `safe-apps` to reduce the scope of the PR. It should be only about the `chains` endpoint

Reply:
Reverted here: 940bc16

Reply:
You can go with url path versioning https://www.django-rest-framework.org/api-guide/versioning/#urlpathversioning

Reply:
> You can go with url path versioning https://www.django-rest-framework.org/api-guide/versioning/#urlpathversioning

But what about the root `urls.py`? What value would the two paths here have for the `namespace`?

```python
path("api/v1/", include("safe_apps.urls", namespace="v1")),
path("api/v1/", include("chains.urls", namespace="v1")),
```

Reply:
I can imagine something like:

```python
path("api", include("safe_apps.urls", namespace="safe-apps")),
path("api", include("chains.urls", namespace="chains")),
```

and inside each app do the versioning. Was this your suggestion @Uxio0 ?

Reply:
If you use url inline you can do:

```python
urlpatterns = [
    re_path(
        r'^(?P<version>(v1|v2))/$',
        include("safe_apps.urls", namespace="safe-apps"),
    ),
    re_path(
        r'^(?P<version>(v1|v2))/$',
        include("chains.urls", namespace="chains"),
    )
]
```

Or you can do. I might prefer this version, looks cleaner:
```python
path("api/v1/", include("safe_apps.urls", namespace="v1:safe-apps")),
path("api/v1/", include("chains.urls", namespace="v1:chains")),
```

For the relay service I couldn't make it work and that's why I have everything on the same `urls` file (not recommended at all 🙈 )

Reply:
> The other option, I might prefer this version, looks cleaner:
> 
> ```python
> path("api/v1/", include("safe_apps.urls", namespace="v1:safe-apps")),
> path("api/v1/", include("chains.urls", namespace="v1:chains")),
> ```

If we go with this, and if eventually a new version gets introduced for example for `safe-apps` then it would be:

```python
path("api/v1/", include("safe_apps.urls", namespace="v1:safe-apps")),
path("api/v2/", include("safe_apps.urls", namespace="v2:safe-apps")),
path("api/v1/", include("chains.urls", namespace="v1:chains")),
```

right? 🤔

Reply:
I went with the first option that you suggested here @Uxio0  https://github.com/gnosis/safe-config-service/pull/88#discussion_r643234361

Since we don't have any other version besides `v1` I left it without the `re_path`. If a new version is added I will update the paths.

Reply:
No, you would go with `path("api/v2/", include("safe_apps.urls_v2", namespace="v2:safe-apps")),` 😉

Reply:
Ah I see. Yeah that works 👍